File: ABAModelDiversity/API/Tables/FirmDemographics.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Tables
{
  [Table("AD_FirmDemographics")]
  public class FirmDemographics
  {
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid FirmDemographicID { get; set; }
    public Guid CompanyProfileID { get; set; }
    public string RegionName { get; set; }
    public string EquityPartners { get; set; }
    public string NonEquityPartners { get; set; }
    public string Associates { get; set; }
    public string Counsel { get; set; }
    public string OtherLawyers { get; set; }
  }
}

File: ABADiversity/src/app/views/surveys/entities/firmdemographics.ts

export interface FDInfo {
    firmId?:string;
    regions?:FDdetails[];
}
export interface FDdetails {
    regionName?:string;
    equityPartners?:string;
    nonEquityPartners?:string;
    associates?:string;
    counsel?:string;
    otherLawyers?:string;
}